src/components/SelectInput/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+
+import { BsCaretDownFill } from "react-icons/bs"
+import Select, {
+  components,
+  DropdownIndicatorProps,
+  PlaceholderProps
+} from "react-select"
+
+import { SelectStyled } from "./styles"
+import { Props } from "./types"
+
+export const SelectInput = ({
```

Comment:
Atribui um alias pro select da lib -> `import ReactSelect`
```suggestion
export const Select = ({
```